packages/babel-types/scripts/utils/stringifyValidator.js

```diff
@@ -46,7 +61,7 @@ export default function stringifyValidator(validator, nodePrefix) {
             return (
               shapeKey +
               (isOptional ? "?: " : ": ") +
-              stringifyValidator(propertyDefinition.validate)
+              stringifyValidator(propertyDefinition.validate, nodePrefix)
```

Comment:
This is a fix. `nodePrefix` should be passed through all `stringifyValidator` calls, although we don't have `shapeOf({ a: assertOfNodeType("...") })` usage.